Convert to ES US

gary-ovh opened the pull request:
Adjusted links, Dates, OVH to OVHcloud

gary-ovh committed:
Convert to ES US

Adjusted links, Dates, OVH to OVHcloud

Zenamine commented on pages/account/customer/phishing_care/guide.es-us.md:
```diff
@@ -74,4 +74,4 @@ Conéctese al [área de cliente de OVH](https://www.ovh.com/auth/?action=gotoman
 
-Interactúe con nuestra comunidad de usuarios en [ovh.es/community](https://www.ovh.es/community/){.external}.
\ No newline at end of file
+Interactúe con nuestra comunidad de usuarios en [ovh.es/community](https://www.ovh.es/community/){.external}.
```
Could you please change these links for the right onese

gary-ovh committed:
Merge remote-tracking branch 'upstream/develop' into develop

gary-ovh committed:
Updates

Fixed community link in phishing guide.
Added 2 screenshots to replaced ones in 2fa guide
